src/sentry/api/endpoints/project_rules_configuration.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import absolute_import
+
+
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.rules import rules
+from rest_framework.response import Response
+
+
+class ProjectRulesConfigurationEndpoint(ProjectEndpoint):
+    def get(self, request, project):
```

Comment:
since the django view's permission was `required_scope = 'project:write'`, i think that to maintain that, you'd want to define the permissions class for this endpoint as `StrictProjectPermission` which doesn't allow `GET` for `project:read`

Reply:
👍 good catch